aries-site/src/examples/templates/filtering/QuickFilterToolbar.js

```diff
@@ -0,0 +1,105 @@
+import React, { useContext } from 'react';
+import {
+  Data,
+  DataTable,
+  DataContext,
+  DataClearFilters,
+  DataFilter,
+  DataFilters,
+  DataSearch,
+  DataSummary,
+  SelectMultiple,
+  Heading,
+  Page,
+  PageContent,
+  Toolbar,
+  Pagination,
+} from 'grommet';
+import applications from '../../../data/mockData/applications.json';
+
+const properties = {
+  title: { filter: false },
+  publisher: { label: 'Publisher' },
+  delivery: { label: 'Delivery', filter: false, badge: false },
+  pricing: { label: 'Pricing' },
+  rating: { label: 'Rating', range: { min: 0, max: 5 } },
+};
+
+const columns = [
+  {
+    property: 'title',
+    header: 'Title',
+    primary: true,
+  },
+  {
+    property: 'publisher',
+    header: 'Publisher',
+    size: 'small',
+  },
+  {
+    property: 'delivery',
+    header: 'Delivery',
+  },
+  {
+    property: 'pricing',
+    header: 'Pricing',
+  },
+  {
+    property: 'rating',
+    header: 'Rating',
+  },
+];
+
+export const QuickFilterToolbar = () => (
+  <Page>
+    <PageContent>
+      <Heading level={2} margin="none">
+        Storage
+      </Heading>
+
+      <Data data={applications} properties={properties}>
+        <ComposedToolbar />
+        <DataSummary />
+        <DataTable aria-describedby="storage-heading" columns={columns} />
+        <Pagination alignSelf="end" />
+      </Data>
+    </PageContent>
+  </Page>
+);
+
+const ComposedToolbar = () => {
+  const { view } = useContext(DataContext);
+  return (
+    <Toolbar gap="medium" align="end">
+      <Toolbar align="end">
+        <DataSearch />
+        <DataFilters updateOn="change" clearFilters={false}>
+          <DataFilter
+            property="delivery"
+            // override HPE theme margin to align with search + filter
```

Comment:
should this not work out of the Box?

Reply:
DataFilter just inherits FormField margin (which is desired when presenting in a filter panel). But if you want to use it standalone you have to remove it.

It'd be ideal not to have to do this, but with how FormField is built at the moment you need to